src/cowrie/output/datadog.py

```diff
@@ -20,9 +21,11 @@
 class Output(cowrie.core.output.Output):
     def start(self):
         self.url = CowrieConfig.get("output_datadog", "url").encode("utf8")
-        self.api_key = CowrieConfig.get("output_datadog", "api_key").encode("utf8")
+        self.api_key = CowrieConfig.get("output_datadog", "api_key", fallback="").encode("utf8")
+        if len(self.api_key) == 0:
+            log.msg("Datadog output module: API key is not defined.")
```

Comment:
you'd likely need to exit here? not only log

Reply:
You can't.